Create ui-c8y-1023-19-0-replace-provided-placeholder-by-empty-string.md (#4095)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1023-19-0-replace-provided-placeholder-by-empty-string.md

---
date: ""

> title: In Software repository $PROVIDED placeholder has been replaced by an empty string value

Reviewer: ```suggestion
title: $PROVIDED placeholder has been replaced by an empty string value in the software repository
```

> title: $PROVIDED placeholder has been replaced by an empty string value in the software repository

Reviewer: ```suggestion
title: Placeholder $PROVIDED has been replaced by an empty string value in the software repository
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5037
version: 1023.19.0
---

> In **Software repository** users can define that a device is responsible for providing the software artifacts itself instead of the user uploading a binary or pointing to it via URL. In this case the created software object in inventory had `$PROVIDED` as a placeholder value for the software `url` property. From now on instead of this placeholder and empty string value will be used. Existing software items in inventory remain unchanged.

Reviewer: ```suggestion
When creating a software item, users can define a device as responsible for providing the software artifacts, rather than uploading a binary or pointing to it via a URL. In this case, the created software object used to have `$PROVIDED` as a placeholder value for the software `url` property in the inventory. With this change, an empty string value will be stored instead. Existing software items in the inventory remain unchanged.
```

---
date: ""

> title: Replace $PROVIDED placeholder by empty string

Reviewer 2: ```suggestion
title: In Software repository $PROVIDED placeholder has been replaced by an empty string value
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5037
version: 1023.19.0
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [feat(Device Management): [DM-5037] Replace $PROVIDED placeholder by
empty
string](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/10510)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[DM-5037]:
https://cumulocity.atlassian.net/browse/DM-5037?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Co-authored-by: Bozhidar Dedov <95280800+bozhidardedov@users.noreply.github.com>

> Co-authored-by: Dominik Schmitz <dominik.schmitz@cumulocity.com>

Reviewer 2: ```suggestion
In **Software repository** users can define that a device is responsible for providing the software artifacts itself instead of the user uploading a binary or pointing to it via URL. In this case the created software object in inventory had `$PROVIDED` as a placeholder value for the software `url` property. From now on instead of this placeholder and empty string value will be used. Existing software items in inventory remain unchanged.
```
